pkg/tui/pages/source_configure/trufflehog_configure.go

```diff
@@ -49,7 +50,7 @@ func GetTrufflehogConfiguration() truffleCmdModel {
 		Key:         "concurrency",
 		Required:    false,
 		Help:        "Number of concurrent workers.",
-		Placeholder: "1",
+		Placeholder: strconv.Itoa(runtime.NumCPU()),
```

Comment:
which was the default concurrency before we added the tui stuff?

Reply:
The updated version. We copied it from here: https://github.com/trufflesecurity/trufflehog/blob/ed062178622c74167551eb8f2322c56a78d53a1d/main.go#L47